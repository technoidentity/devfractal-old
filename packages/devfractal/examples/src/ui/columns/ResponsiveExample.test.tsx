import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Responsive } from './ResponsiveExample'

test('Responsive Example of columns', () => {
  const component: ReactTestRenderer = renderer.create(<Responsive />)
  expect(component.toJSON()).toMatchSnapshot()
})
