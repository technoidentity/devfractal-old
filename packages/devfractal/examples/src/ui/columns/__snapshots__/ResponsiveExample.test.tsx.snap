// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Responsive Example of columns 1`] = `
<div>
  <section
    className="section"
  >
    <div
      className="container"
    >
      <h1
        className="title is-4"
      >
        Mobile columns
      </h1>
      <div
        className="columns is-mobile-only"
      >
        <div
          className="column"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            1
          </div>
        </div>
        <div
          className="column"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            2
          </div>
        </div>
        <div
          className="column"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            3
          </div>
        </div>
        <div
          className="column"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            4
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    className="section"
  >
    <div
      className="container"
    >
      <h1
        className="title is-4"
      >
        Desktop columns
      </h1>
      <div
        className="columns is-desktop-only"
      >
        <div
          className="column"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            1
          </div>
        </div>
        <div
          className="column"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            2
          </div>
        </div>
        <div
          className="column"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            3
          </div>
        </div>
        <div
          className="column"
        >
          <div
            className="notification is-primary has-text-centered"
          >
            4
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
